### ISSUE TITLE
Unable to fetch custom packages from pypi-local on artifactory

### ISSUE THREAD

**Issue by amang95** [willemt/pip-versions#2]
Hey

Until two weeks ago, i was able to successfully execute `pip-versions list package-name` where i was presented with all the versions of the package maintained within pypi-local on artifactory.
Now when I try doing that, it returns no output.
I can confirm that the package exists and can download it using `pip download package-name`.

Also I noticed however that `pip-versions` is able to list versions for official pypi packages like numpy, pandas etc.

Need some clarity on this as this has resulted in quite some broken scripts.

**Comment by willemt** [comment]
sorry to hear that @amang95

Can you please send me your pip version? `ie. pip --version`

I have a hunch the API pip-versions is using has broken

**Comment by amang95** [comment]
Pip version: `pip 22.0.4 from /usr/local/lib/python3.7/site-packages/pip (python 3.7)`

**Comment by willemt** [comment]
hi @amang95 I checked out this version but can't reproduce it sorry. I've also tried reproducing with my private pypi installation but no luck reproducing it. I have a feeling this might be an artifactory side issue. I need more debugging info to proceed since I don't use artifactory.

I'll try to add some debug logging sometime to further pin down what could be going wrong.

**Comment by amang95** [comment]
Yes some logs could be helpful too. Right now the command just doesn't print anything.
